src/main/java/com/greedobank/cards/MyConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.greedobank.cards;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+@PropertySource("classpath:application.properties")
+public class MyConfig {
```

Comment:
I believe even `@Configuration`-annotated classed should not be at the package root of Spring's project

Reply:
Done